index.js

```diff
@@ -534,7 +534,16 @@ Command.prototype.executeSubCommand = function(argv, args, unknown) {
     proc = spawn(process.execPath, args, { stdio: 'inherit'});
   }
 
-  proc.on('close', process.exit.bind(process));
+  //killing the child process if the main process is terminated
+  var processClosing = function(){
+    proc.kill('SIGHUP');
+  }
+  process.on('exit',processClosing );
+  process.on('SIGINT', processClosing );
+  process.on('SIGHUB', processClosing );
```

Comment:
I fixed it. Now the exit signal of the main process is used for the sub command, too.